tests/test_base.py

```diff
@@ -296,7 +310,7 @@ def test_get_token_file_different_name(self):
         token_in_file = str(uuid4())
         with self.get_token_as_file(token_in_file) as filename:
             config = {"mytoken_file": filename}
-            self.assertEqual(
+            self.assertEqual(  # nosec
```

Comment:
I think this slipped in from your other PR https://github.com/psss/did/pull/297 .
```suggestion
            self.assertEqual(
```